### DISCUSSION TITLE
How to handle logging?

### DISCUSSION THREAD

**Discussion by LiveNathan** [tjake/Jlama#113] [Q&A]
Hi tjake! How do recommend I handle logging?

I saw the use of [logback in your example](https://github.com/langchain4j/langchain4j-examples/blob/main/jlama-examples/src/main/resources/logback.xml) but I don't see any logs in the console.

**Comment by tjake** [comment]
If you switch to debug logs you should see something?

**Reply by LiveNathan** [reply]
Thanks! I'll give it a chat when I wrap the current branch.
